docs/ui/components/Collapsible/index.tsx

```diff
@@ -40,7 +40,7 @@ const Collapsible: ComponentType<CollapsibleProps> = withHeadingManager(
     open = false,
   }: CollapsibleProps & HeadingManagerProps) => {
     // track open state so we can collapse header if it is set to open by the URL hash
-    const [isOpen, setOpen] = useState<boolean>(open);
+    const [isOpen, setIsOpen] = useState<boolean>(open);
```

Comment:
What was the intent with the rename? This seems unrelated to the issue.

Reply:
Mostly the destructured value we see in codebase in form of `[value, setValue]` so thought of improving it here itself since it lies in the same component. If still seems unrelatable I can revert it. Please advise.

Thanks

Reply:
Thanks for the context! We can leave it as-is, since we have no strict patterns defined on naming the state methods.